azure-pipeline.nightly.yml

```diff
@@ -31,6 +31,8 @@ extends:
 
     l10nSourcePaths: ./src
 
+    nodeVersion: "20.x"
```

Comment:
Adding these to make sure we're using a consistent node version across pipelines. The default in the template was already node 20